### PR TITLE
Move to uniform 

### PR DESCRIPTION
**Pull request by alshakero**
Auth is disabled to make testing easier. Once you approve I'll reenable it.

### CODE REVIEW

File: src/Content/wwwroot/Content/views/DialogPage.html [hunk 1]
```diff
@@ -26,8 +26,8 @@
                 <h3 class="content-dialog-page__text">You have unsaved changes.</h3>
                 <h3 class="content-dialog-page__text">Are you sure you want to continue?</h3>
                 <div class="content-dialog-page__buttons">
-                    <button type="button" class="btn btn-lg btn-default content-dialog-page__button--dismissable" on-click="onRejectClick" dialog-dismiss>No</button>
-                    <button type="submit" class="btn btn-lg btn-default"
+                    <button type="button" on-click="onRejectClick" dialog-dismiss>No</button>
```
Comment (Mihaiii) [inline src/Content/wwwroot/Content/views/DialogPage.html]: Why was `content-dialog-page__button--dismissable` removed?

File: src/Content/wwwroot/Content/views/EditorPage.html [hunk 2]
```diff
@@ -62,23 +62,23 @@
                     }
             </style>
 
-            <label slot="content/form-name-label" class="control-label">Name:</label>
+            <label slot="content/form-name-label">Name:</label>
```
Comment (Mihaiii) [inline src/Content/wwwroot/Content/views/EditorPage.html]: Why was `control-label` removed? (same question for all `control-label`s)

**Comment by Mihaiii** [comment]
@alshakero : I have more of a general question starting from a particular case in Content app.
I have this view:

btn

Now, for this case, the "GO!" button is too wide (due to uniforms' `min-width: 120px` rule). I understand that in general the purpose is to have "a common look and feel" when multiple apps are used, so should I leave it as is or apply my own `min-width` rule? Where is the border between "leave it as uniform provides it" and "apply your own custom rules with higher priority than uniforms'"?

**Comment by alshakero** [comment]
In such case you can use `uni-flat-button`.

**Pull request merged**